content/source/layouts/telefonicaopencloud.erb

```diff
@@ -0,0 +1 @@
+../../../ext/providers/PROVIDER/website/telefonicaopencloud.erb
```

Comment:
Should be:
../../../ext/providers/telefonicaopencloud/website/telefonicaopencloud.erb